source/docs/v3/quickstart.blade.md

```diff
@@ -136,15 +136,6 @@ Now we need to actually specify the central domains. A central domain is a domai
 ],
 ```
 
-If you're using Laravel Sail, no changes are needed, default values are good to go:
-
-```php
-'central_domains' => [
-    '127.0.0.1',
-    'localhost',
-],
-```
```

Comment:
I think this can be added back. The new sail page doesn't address this at all so there's no reason to remove it.